tests/app/views/contexts/summary/test_group.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+
+from app.questionnaire.location import Location
+from app.views.contexts.summary.group import Group
+
+
+@pytest.fixture(name='schema')
+def fixture_schema():
+    return {
+        'blocks': [
+            {
+                'type': 'Question',
+                'id': 'number-question-one',
+                'question': {
+                    'answers': [
+                        {
+                            'id': 'answer-one',
+                            'mandatory': False,
+                            'type': 'Number',
+                            'label': 'Leave blank',
+                            'default': 0,
+                        }
+                    ],
+                    'id': 'question',
+                    'title': "Don't enter an answer. A default value will be used",
+                    'type': 'General',
+                },
+            },
+            {
+                'type': 'Question',
+                'id': 'number-question-two',
+                'show_on_section_summary': False,
+                'question': {
+                    'answers': [
+                        {
+                            'id': 'answer-two',
+                            'mandatory': False,
+                            'type': 'Number',
+                            'label': 'Enter a Value',
+                        }
+                    ],
+                    'id': 'question2',
+                    'title': 'Enter an answer. It will be shown on the summary page',
+                    'type': 'General',
+                },
+            },
+            {'type': 'Summary', 'id': 'summary'},
+        ],
+        'id': 'group',
+        'title': 'group-test',
+    }
+
+
+@pytest.fixture(name='path')
+def fixture_path():
+    return [
+        Location('section', 'number-question-one'),
+        Location('section', 'number-question-two'),
+    ]
+
+
+def test_group_serialize_hide_on_summary_true(
+    app, mocker, schema, path
+):  # pylint: disable=unused-argument
+
+    answer_store = mocker.MagicMock()
```

Comment:
Can `MagicMock` from `mock` not be used?

Reply:
Yes, as the MagicMock seen here _is_ a MagicMock from stdlib. However, if we have a long term goal of moving away from unittest to pytest, we should embrace where possible and sensible pytest plugins that can give cleaner test code. See https://github.com/pytest-dev/pytest-mock/ for examples of where the mocker fixture may help, especially when patching.

Reply:
Does this module actually make the code in the PR any cleaner over and above mock?

I don't think we should be introducing dependencies if we're not actually using the features they provide.

Reply:
Having the mock available as a common fixture across unit tests _is_ using part of the functionality they provide.

Reply:
I think if we want to change the approach to mocks, we should do it across all tests in a separate pull request. Otherwise it's confusing when looking at the code base why it's done differently.

Reply:
Done :+1: